src/components/dev-hub/podcast-jumbotron.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+
+import { H2, P } from '~components/dev-hub/text';
+import BlogTagList from '~components/dev-hub/blog-tag-list';
+import BylineBlock from '~components/dev-hub/byline-block';
+import HeroBanner from '~components/dev-hub/hero-banner';
+
+import {
+    colorMap,
+    fontSize,
+    screenSize,
+    size,
+} from '~components/dev-hub/theme';
+
+const Title = H2.withComponent('h1');
+
+const PostMetaLine = styled('div')`
+    color: ${colorMap.greyLightThree};
+    display: flex;
+    font-size: ${fontSize.tiny};
+    margin: ${size.medium} 0 40px;
+
+    @media ${screenSize.upToLarge} {
+        flex-direction: column;
+        font-size: ${fontSize.xsmall};
+        margin: ${size.default} 0 ${size.medium};
+    }
+`;
+
+const DateText = styled(P)`
+    margin-right: ${size.tiny};
+
+    @media ${screenSize.upToLarge} {
+        font-size: ${fontSize.xsmall};
+    }
+`;
+
+const DateTextContainer = styled('div')`
+    display: flex;
+    margin-right: ${size.medium};
+`;
+
+const PodcastJumbotron = ({
+    breadcrumb,
+    image,
+    publishDate,
+    title,
+    tags, //TODO: In Progress
+    authors, //TODO: In Progress
+}) => (
+    <HeroBanner background={image} breadcrumb={breadcrumb}>
+        <Title collapse>{title}</Title>
+        <PostMetaLine>
+            <DateTextContainer>
+                {publishDate && (
+                    <DateText collapse>Published: {publishDate}</DateText>
+                )}
```

Comment:
The format is slightly different here than it is for articles, where we have `MM DD, YYYY`. Would it make sense to use that format here as well?

Reply:
fixed